src/Ombi/ClientApp/src/app/requests-list/components/grid-spinner/grid-spinner.component.scss

```
@@ -1,12 +1,74 @@
-  top: 0;
-  left: 0;
-  bottom: 56px;
-  right: 0;
-  background: rgba(0, 0, 0, 0.15);
-  z-index: 1;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.6.0)

[error] Unexpected deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)